cmd/resim/commands/batch.go

```diff
@@ -65,6 +65,11 @@ var (
 	}
 )
 
+const (
+	GITHUB_ACTOR_KEY  = "GITHUB_ACTOR"
```

Comment:
Looks right to me 👍 I think the alternative `GITHUB_TRIGGERING_ACTOR` wouldn't really give us quite what we want